src/components/footer/index.tsx

```diff
@@ -15,10 +18,48 @@ function Footer({
 }: FooterProps): React.ReactElement {
   return (
     <footer className={classNames(s.root, className)}>
-      Dev Portal Footer Placeholder{' '}
-      {typeof openConsentManager == 'function' && (
-        <button onClick={openConsentManager}>Open Consent Manager</button>
-      )}
+      <InlineSvg className={s.logo} src={svgHashicorpLogo} />
+      <ul className={s.links}>
+        <li className={s.linkListItem}>
```

Comment:
For consideration: many of the other places where we render links like this, the link data is declared as JSON/data somewhere. Do we think it might make sense to do the same for the footer links?

Reply:
I think it could, but held off for now due to the `Cookie manager` button wedged in the middle. Definitely open to coming up with a data structure that accounts for that if we feel this is a priority 👍